step03/db.py

```diff
@@ -0,0 +1,68 @@
+from flask import g
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import create_engine
+import models
+from models import Profiles
+
+Base = models.Base
+
+engine = create_engine('sqlite+pysqlite:///imagely.db')
+Base.metadata.create_all(engine)
+Session = sessionmaker(bind=engine)
+session = Session()
+
+
+def registered_user(github_id):
```

Comment:
I also think it makes more sense to check whether they're registered using username or email since it's what you're using to uniquely identify your users.